packages/e2e-tests/helpers/index.ts

```diff
@@ -83,3 +87,9 @@ export async function startTest(page: Page, example: string, apiKey?: string) {
     await page.locator('[data-test-id="ViewToggle-DevTools"]').waitFor();
   }
 }
+
+export async function startTest(page: Page, example: string, apiKey?: string) {
+  const recordingId = exampleRecordings[example];
+
+  openRecording(page, example, recordingId, apiKey);
+}
```

Comment:
What's the value of splitting `openRecording` and `startTest` into separate functions?

I think we should merge them back into a single function. It's not clear why or when you would use one vs the other currently.

Reply:
Initially kept separate to allow opt-in rather than refactoring all the tests. Now that cloned API seems to be settled, keeping just `startTest`